### DISCUSSION TITLE
embed Math Notepad?

### DISCUSSION THREAD

**Discussion by lochiwei** [josdejong/mathjs#2201] [Q&A]
Can we embed [Math Notepad](https://mathnotepad.com/docs/index.html) on other websites, say like [Observable](https://observablehq.com/)?

**Comment by josdejong** [comment]
There are some options for that indeed: menu File -> Share. Is that what you mean?

**Reply by lochiwei** [reply]
it seems the Share option doesn't work on [Observable](https://observablehq.com/@lochiwei/math-notepad), could you take a look for me? I've made some efforts to try to get it work, but in vain.

**Reply by josdejong** [reply]
I'm not familiar with Observable. I have no idea if it supports loading a script like `<script src="https://mathnotepad.com/embed.min.js" async></script>`, it could very well be that it has limitations in that regard.

**Reply by dvd101x** [reply]
Hi! I recently implemented a way to embed mathjs directly on observable markdown, html and cells.

please check https://observablehq.com/@dvd101x/mathjs-inside-html-and-markdown

**Reply by josdejong** [reply]
That is cool 😎 , thanks for sharing David!
